GHzDACs/Cleanup/dac.py

```diff
@@ -1064,16 +1064,18 @@ def cmdTime_cycles(cmd):
         if opcode in [0x0, 0x1, 0x2, 0x4, 0x8, 0xA]:
             return 1
         #branch to start
-        if opcode == 0xF:
+        elif opcode == 0xF:
             return 2
         #delay
-        if opcode == 0x3:
+        elif opcode == 0x3:
             return MemorySequence.getAddress(cmd) + 1
         #run sram
-        if opcode == 0xC:
+        elif opcode == 0xC:
             # TODO: Incorporate SRAMoffset when calculating sequence time.
             #       This gives a max of up to 12 + 255 us
             return 25*12 # maximum SRAM length is 12us, with 25 cycles per us
+        else:
+            raise Exception("Unkown opcode")
```

Comment:
"Unknown"

Also, I would suggest including the bad opcode here:

```python
raise Exception("Unknown opcode: {}".format(opcode))
```

Otherwise, LGTM